code/extensions/mangle-loader.js

```diff
@@ -25,7 +25,7 @@ function getMangledFileContents(projectPath) {
 	if (!entry) {
 		const log = (...data) => fancyLog(ansiColors.blue('[mangler]'), ...data);
 		log(`Mangling ${projectPath}`);
-		const ts2tsMangler = new Mangler(projectPath, log, { mangleExports: true, manglePrivateFields: true });
+		const ts2tsMangler = new Mangler(projectPath, log, { mangleExports: false, manglePrivateFields: true });
 		entry = ts2tsMangler.computeNewFileContents();
```

Comment:
Sorry, could you explain why we should change default VS Code behaviour here?

Reply:
Because the mangler fails for che-api extension: https://github.com/eclipse/che/issues/22296#issuecomment-1597383783
Even when the exports are annotated with `@skipMangle`.
If we can make time for it, I filed [the issue](https://github.com/eclipse/che/issues/22337) to dig into the implementation details of VS Code mangling process and investigate it.